tp_timesheet/docker_handler.py

```
@@ -29,6 +30,14 @@ def pull_image(self):
     def run_container(self):
         """ Run the slenium image in a container """
         self.container = self.client.containers.run(self.image, auto_remove=True, detach=True, shm_size='2g', ports={'4444/tcp':4444})
+        # wait for container to run
+        timeout = 5
+        stop_time = 0.5
+        elasped_time = 0
+        while self.container.status != 'running' and elasped_time < timeout:
+            sleep(stop_time)
+            elasped_time += stop_time
+            continue
```

Should throw an error if the timeout elapses. Currently it will continue and likely crash

---

Might be worth throwing `container.reload()` into the loop just incase the status doesn't update?